Formula/hyperkit.rb

```diff
@@ -0,0 +1,85 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit/archive/v0.20180403.tar.gz"
+  sha256 "e2739b034f20d9437696de48ace42600f55b7213292ec255032b2ef55f508297"
+
+  depends_on "aspcud" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
+  depends_on :xcode => ["9.0", :build]
+
+  depends_on "libev"
+
+  resource "tinycorelinux" do
+    url "https://dl.bintray.com/markeissler/homebrew/hyperkit-kernel/tinycorelinux_8.x.tar.gz"
+    sha256 "560c1d2d3a0f12f9b1200eec57ca5c1d107cf4823d3880e09505fcd9cd39141a"
+  end
+
+  def install
+    system "opam", "init", "--no-setup"
+    opam_dir = "#{buildpath}/.brew_home/.opam"
+    ENV["CAML_LD_LIBRARY_PATH"] = "#{opam_dir}/system/lib/stublibs:/usr/local/lib/ocaml/stublibs"
```

Comment:
Don't assume `/usr/local`. You'll want to use:
```ruby
Formula["ocaml"].opt_lib/"ocaml/stublibs"
```
here I guess, or some variation thereof.